src/sinks/clickhouse/native/convert.rs

```diff
@@ -10,14 +10,15 @@ use std::{
     sync::Arc,
 };
 use value::Value;
+use vector_core::event::Event;
 
 pub(super) fn build_block(
     schema: Vec<(String, SqlType)>,
-    events: Vec<LogEvent>,
+    events: Vec<Event>,
 ) -> crate::Result<Block> {
     let mut b = Block::new();
     for ev in &events {
-        b.push(get_row_from_events(&schema, ev)?)
+        b.push(get_row_from_events(&schema, &ev.clone().into_log())?)
```

Comment:
It _seems_ like from the usage of `build_block` that the `events` arg could take ownership and that this clone wouldn't be necessary. Perhaps remove the `&` from the line above? Or am I overlooking something.

Reply:
Ok. I have change the `get_row_from_events`'s `event` type to take ownership as well. Fixed.